app/controllers/api/v1/passwords_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Api::V1
+  class PasswordsController < Devise::PasswordsController
+
+    api :POST, '/users/password', 'Reset password'
+    def create
+      @user = User.find_by_email(user_params[:email])
```

Comment:
@yunixon use `find_by!(email: ...)`. If there is no such user there will be `RecordNotFound` exception which is processed by standard handler as 404 error. Btw `find_by_...` is deprecated.